commons-vfs2/src/test/java/org/apache/commons/vfs2/provider/hdfs/test/HdfsFileProviderTest.java

```diff
@@ -117,7 +117,7 @@ public static void setUmask(final Configuration conf2) {
             final BufferedReader bri = new BufferedReader(new InputStreamReader(p.getInputStream()));
             final String line = bri.readLine();
             p.waitFor();
-            final Short umask = Short.parseShort(line.trim(), 8);
+            final short umask = Short.parseShort(line.trim(), 8);
```

Comment:
`Short.parseShort(line.trim(), 8)` -> `Short.parseShort(line.trim(), 8).shortValue()` to make unboxing explicit

Reply:
Don't fallow. Revert only this change